vita3k/app/src/app_init.cpp

```diff
@@ -104,83 +104,125 @@ void update_viewport(EmuEnvState &state) {
 
 void init_paths(Root &root_paths) {
     auto sdl_base_path = SDL_GetBasePath();
-    auto sdl_pref_path = SDL_GetPrefPath(org_name, app_name);
     auto base_path = fs_utils::utf8_to_path(sdl_base_path);
-    auto pref_path = fs_utils::utf8_to_path(sdl_pref_path);
     SDL_free(sdl_base_path);
-    SDL_free(sdl_pref_path);
+
     root_paths.set_base_path(base_path);
-    root_paths.set_pref_path(pref_path);
-    root_paths.set_log_path(base_path);
-    root_paths.set_config_path(base_path);
     root_paths.set_static_assets_path(base_path);
-    root_paths.set_shared_path(base_path);
-    root_paths.set_cache_path(base_path / "cache" / "");
+
+#if defined(__APPLE__)
+    // On Apple platforms, base_path is "Contents/Resources/" inside the app bundle.
+    // An extra parent_path is apparently needed because of the trailing slash.
+    auto portable_path = base_path.parent_path().parent_path().parent_path().parent_path() / "portable" / "";
```

Comment:
https://wiki.libsdl.org/SDL2/SDL_GetBasePath

SDL_GetBasePath's behaviour can be overridden by adding a property to the Info.plist file

Reply:
I learned about that before, but we actually do still need the Resources path for the static assets and update script, and if I change the SDL behavior to give the directory the bundle is in, we won’t know what the bundle is called to get the Resources directory.

Open to any ideas on how to deal with that if you want.

Reply:
Ah okay. Then using current implentation for a while and chaing SDL_GetBasePath's behaviour when deprecating 'Contents/Resources' usage would be better.